ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -527,6 +715,42 @@ def initialize_session(
         """
         return self._initialize_session_core(session_constructor, instrument_type_id)
 
+    def initialize_multiplexer_session(
```

Comment:
Please add a feature toggle for any new APIs that you are introducing, so that they won't be enabled until the feature is complete. This includes the APIs you have already added.

You can do this in a separate PR, but please do it before the 1st 1.4 prerelease build.

Reply:
Sure. Since this PR already holds numerous changes. I'll take that under this [work item](https://dev.azure.com/ni/DevCentral/_workitems/edit/2649584) right after closing this PR.